app/views/unattended/provisioning_templates/snippet/pxegrub2_chainload.erb

```diff
@@ -43,7 +44,7 @@ echo "the grub2_chainload template."
 echo
 echo "(*) grub2-efi-x64-2.02-122.el8 (upstream doesn't have the patches yet)"
```

Comment:
Any idea about Debian/Ubuntu?

Reply:
They are using >= 2.06 of grub2:
https://packages.debian.org/search?suite=buster&searchon=names&keywords=grub2

Reply:
But does that have the patches? If I look in the grub source (https://git.savannah.gnu.org/git/grub.git) I don't see the word `connectefi` at all. https://src.fedoraproject.org/rpms/grub2/blob/rawhide/f/0205-efi-new-connectefi-command.patch adds in for Fedora and I assume the same patch is applied in EL8.

If I look at https://udd.debian.org/patches.cgi?src=grub2&version=2.06-13 then I don't see anything like it, but I may also be missing something.

Reply:
Ah, its a separate patch. Unfortunately, the patch does not exist in debian/ubuntu.
Should we use something like "if not running on debian/ubuntu then...."?

Reply:
I think this depends on the grub version that is installed on the Smart Proxy (TFTP proxy to be precise), which is rather hard to determine. I suppose that's why it was a comment.

If you want to do it reliably, perhaps it should be a setting/capability on the Smart Proxy TFTP feature that gets exposed and then the template reads. It's quite a bit of plumbing, but I'm unsure about the alternatives.

Reply:
Ok. I think it does make sense to enable this feature by default and mention it in the template itself, that it needs to be commented out on Debian/Ubuntu. I guess, most of the foreman systems are running on Fedora/EL and then this feature would work. What do you think?

Reply:
@ekohl What are your thoughts?

Reply:
Hard to say. I wonder if a global parameter would make sense, so users don't need to modify the template. As for the question if it should be opt-in or opt-out, I believe the majority of users is on EL so we can make it opt-out. Probably something we should document though.